src/NHibernate/Util/ExpressionsHelper.cs

```diff
@@ -15,5 +17,40 @@ public static class ExpressionsHelper
 			}
 			return ((MemberExpression)expression.Body).Member;
 		}
+
+		internal static string TryGetEntityName(ISessionFactoryImplementor sessionFactory, MemberExpression memberExpression, out string memberPath)
```

Comment:
Copied from #2079. Added as a static helper method so that can be used by other expression visitors.